src/unity/toolkits/supervised_learning/supervised_learning.cpp

```diff
@@ -519,6 +519,14 @@ gl_sarray supervised_learning_model_base::fast_predict(
   } else if (output_type == "probability_vector"){
     ret_type = flex_type_enum::VECTOR;
   } else {
+      if (output_type == "probability" && variant_get_value<size_t>(state.at("num_classes")) > 2) {
```

Comment:
Nit: Let's match the surrounding code and only indent by 2 here, not 4.

(We sometimes use an indent level of 4 for code that is essentially wrapped due to line length)